client/js/components/header/burger.js

```diff
@@ -0,0 +1,61 @@
+import showHide from './../show-hide';
+import { KEYS } from './../../util';
+
+const headerBurger = (el) => {
+  const headerLower = el.querySelector('.js-header-lower');
+  const burger = showHide({el: headerLower});
+
+  const init = () => {
+    setBurgerAria(isBurgerVisible());
+    handleEvents();
+  };
+
+  const setBurgerAria = (value) => {
+    if (value) {
+      burger.el.setAttribute('aria-expanded', burger.getActive());
+    } else {
+      burger.el.removeAttribute('aria-expanded');
+    }
+  };
+
+  const isBurgerVisible = () => {
+    const isDisplayed = window.getComputedStyle(burger.trigger)
```

Comment:
I've completely missed this - we aren't showing our navigation on mobile?
That seems a little odd no? Especially as we have 5 items.

Either way, I am going to keep UX conversations out of this for now as we can discuss after.